com.github.yucefsourani.pysavetube.json

```diff
@@ -0,0 +1,115 @@
+{
+    "app-id" : "com.github.yucefsourani.pysavetube",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "40",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "pysavetube.py",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=home",
+        "--device=dri",
+        "--socket=pulseaudio"
+    ],
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name" : "intltool",
+            "cleanup" : [
+                "*"
+            ],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://launchpad.net/intltool/trunk/0.51.0/+download/intltool-0.51.0.tar.gz",
+                    "sha256" : "67c74d94196b153b774ab9f89b2fa6c6ba79352407037c8c14d5aeb334e959cd"
+                }
+            ]
+        },
+    {
+      "name": "libsass",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sass/libsass/archive/3.6.4.tar.gz",
+          "sha512": "1f79553be31949d45adf440dcbcf4b828a7e4f71c6f63f288f99451fccc490ebf0c68c58d91deabede37962e35a8cf9af8b0d60d89349fd0adea1ac061c8c2c8"
+        },
+        {
+          "type": "script",
+          "dest-filename": "bootstrap",
+          "commands": ["autoreconf -fi"]
+        }
+      ]
+    },
+    {
+      "name": "sassc",
+      "buildsystem": "autotools",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/sass/sassc/archive/3.6.1.tar.gz",
+          "sha256": "8cee391c49a102b4464f86fc40c4ceac3a2ada52a89c4c933d8348e3e4542a60"
+        },
+        {
+          "type": "script",
+          "dest-filename": "bootstrap",
+          "commands": ["autoreconf -fi"]
+        }
+      ]
+    },
+	{
+	  "name" : "libadwaita",
+	  "buildsystem" : "meson",
+	  "config-opts": [
+	    "-Dexamples=false",
+	    "-Dtests=false"
+	  ],
+	  "sources" : [
+	    {
+	      "type" : "git",
+	      "url" : "https://gitlab.gnome.org/GNOME/libadwaita.git",
+	      "branch" : "main"
```

Comment:
please use a tag or a commit, not a branch. Branches move and then the build is not reproducible.

Reply:
main branch is old version i use gtk3 + libhandy,  but gtk_libadwaita branch i use gtk4 + libadwaita .
Since the version of Libadwaita is still Alpha, this branch "gtk_libadwaita" will continue for a long time .
(google trasnlate)